inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/RecommendationEditorExtension.java

```diff
@@ -245,6 +250,12 @@ private void actionRejectRecommendation(AnnotationActionHandler aActionHandler,
     public void render(CAS aCas, AnnotatorState aState, VDocument aVDoc,
                        int aWindowBeginOffset, int aWindowEndOffset)
     {
+        // do not show predictions during curation or when viewing others' work
+        if (aState.getMode().equals(Mode.CURATION) || 
```

Comment:
Rather than checking for not curation (blacklisting), I would check here whether it is annotation (whitelisting), so that the error does not appear in the next mode we add